lib/stdlib.ngs

```diff
@@ -2105,6 +2105,22 @@ section "Range" {
 	TEST 1 =~ 0..5
 	TEST 0 =~ 0..5
 	TEST 5 !~ 0..5
+
+	doc Checks that x is in the range.
+	doc %EX - ensure(3, 3..10)  # 3
+	F ensure(n:Int, r:NumRange){
+		n in r returns n
```

Comment:
Since you already have `cond` below, it would be better to move this line into `cond`:
```
cond {
  n in r n
  ...
}
```

You can then shorten:
```
F ensure(n:Int, r:NumRange) cond {
  ...
}

Reply:
Interesting, Changed.